file_store/src/file_sink.rs

```diff
@@ -490,16 +514,20 @@ mod tests {
         let tmp_dir = TempDir::new().expect("Unable to create temp dir");
         let (shutdown_trigger, shutdown_listener) = triggered::trigger();
 
-        let (file_sink_client, mut file_sink_server) =
-            FileSinkBuilder::new(FileType::EntropyReport, tmp_dir.path(), "fake_metric")
-                .roll_time(chrono::Duration::milliseconds(100))
-                .create()
-                .await
-                .expect("failed to create file sink");
+        let (file_sink_client, mut file_sink_server) = FileSinkBuilder::new(
+            FileType::EntropyReport,
+            tmp_dir.path(),
+            "fake_metric",
```

Comment:
whats with these 'fake_metric' refs?

Reply:
That is in a test, so the metric name is irrelevant.